### PR TITLE
EIA capacity update

### PR DESCRIPTION
**Pull request by VIKTORVAV99**
## Issue
Some zones have had significant enough changes that it could affect our validation.

## Description

Updates all EIA zones with the latest data.

### Double check

- [x] I have run `pnpx prettier@2 --write .` and `poetry run format` in the top level directory to format my changes.

### CODE REVIEW

**Comment by FelixDQ** [comment]
Great! I'll review this next week

**Comment by jenshoej** [comment]
The capacity data for Solar in Denmark is very outdated. The newest account by the Danish Energy Agency states 3.645 GW solar, whereas the ENTSO-E data you use is 2.3 GW

The report can be found here: https://energinet.dk/om-publikationer/publikationer/redegorelse-for-elforsyningssikkerhed-2023/

**Comment by VIKTORVAV99** [comment]
> The capacity data for Solar in Denmark is very outdated. The newest account by the Danish Energy Agency states 3.645 GW solar, whereas the ENTSO-E data you use is 2.3 GW
> 
> 
> 
> The report can be found here: https://energinet.dk/om-publikationer/publikationer/redegorelse-for-elforsyningssikkerhed-2023/

Please open a separate issue for that, this PR focuses just on EIA data.

**Comment by FelixDQ** [comment]
Hey @VIKTORVAV99, I will be improving our monitoring of the quality validation in the next few weeks, and it would be nice to get that done before merging this, so that I can observe the difference. Would it be okay to wait a few weeks before merging this or do you think that we need it now? :-)

**Review by FelixDQ — APPROVED** [review]
Actually, looking at the changes, I don't think any of this would significantly impact the validation. Let's go ahead and merge it now

File: config/zones/US-CAL-CISO.yaml [hunk 1]
```diff
@@ -11,17 +11,23 @@ capacity:
...
+      source: EIA.gov
+      value: 8836.1
   biomass:
     - datetime: '2023-10-01'
       source: EIA.gov
       value: 993.1
     - datetime: '2023-12-01'
       source: EIA.gov
       value: 970.9
-    - datetime: '2024-03-01'
```
Comment (tonypls) [inline config/zones/US-CAL-CISO.yaml:24]: Is this intentional?

Comment (VIKTORVAV99) [reply config/zones/US-CAL-CISO.yaml:24]: Yes, the script deduplicates the datetimes so only the first recorded datetime with that capacity is kept. When I re-ran it I ran it month by month for this year this updated some of the datetimes as well.

**Pull request merged**